client/api/filesystem/v1alpha1/api.pb.go

```diff
@@ -1,5 +1,5 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
-// source: github.com/kubernetes-csi/csi-proxy/client/api/filesystem/v1alpha1/api.proto
```

Comment:
Is this dependent on where protoc-gen-go is executed from?

Reply:
some external examples: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/storage/v1/generated.pb.go#L18
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/networking/v1/generated.pb.go#L18